.github/workflows/scripts/linux/appimage-qt.sh

```diff
@@ -200,5 +203,5 @@ done
 
 echo "Generating AppImage..."
 rm -f "$NAME.AppImage"
-$APPIMAGETOOL -v "$OUTDIR" "$NAME.AppImage"
+ARCH=x86_64 VERSION=test "$APPIMAGETOOL" -s "$OUTDIR" && mv ./*.AppImage "$NAME.AppImage"
```

Comment:
This version string gets appended into our desktop file, so we probably want this to actually be something relevent

We can probably lift this code from here `generate-metainfo.sh` to match the version string included in the meteinfo file https://github.com/PCSX2/pcsx2/blob/112d02eaf58ef10c549f76eded3894a3679aafe9/.github/workflows/scripts/linux/generate-metainfo.sh#L12-L21

Reply:
Done